lxd/auth/authorization_types.go

```diff
@@ -31,6 +31,9 @@ const (
 	// EntitlementPermissionManager is the `permission_manager` Entitlement. It applies to entity.TypeServer.
 	EntitlementPermissionManager Entitlement = "permission_manager"
 
+	// EntitlementCanViewPermissions is the `can_view_permissions` Entitlement. It applies to entity.TypeServer.
+	EntitlementCanViewPermissions Entitlement = "can_view_permissions"
```

Comment:
the commit message would have benefited from explaining *why* the new entitlement is needed.